atomic_reactor/util.py

```diff
@@ -852,4 +852,145 @@ def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, par
     return dfparser
 
 
+class Labels(object):
```

Comment:
I am not sure that I understand, but this class should provide labels not values, we need to know name of new label, name of old names, name of preferred from given dockerfile